scripts/us_census/geojsons_low_res/plotter.py

```diff
@@ -26,24 +26,21 @@
 matplotlib.use('TkAgg')
 import matplotlib.pyplot as plt
 
-FLAGS = flags.FLAGS
-flags.DEFINE_string('original_path',
-                    default=None,
-                    help='Path to original geojson to be compared.')
-flags.DEFINE_string('simplified_path',
-                    default=None,
-                    help='Path to simplified geojson to be compared.')
-flags.mark_flag_as_required('original_path')
-flags.mark_flag_as_required('simplified_path')
-
 _, (ax1, ax2) = plt.subplots(ncols=2, sharex=True, sharey=True)
 
 
 def compare_plots(geojson1, geojson2, show=True):
```

Comment:
Please add a function comment per https://google.github.io/styleguide/pyguide.html#383-functions-and-methods.

Reply:
Added.